sdk/provisioning/Azure.Provisioning.Storage/tests/BasicStorageTests.cs

```diff
@@ -267,7 +267,7 @@ await test.Define(
                 BlobService blobs = new(nameof(blobs)) { Parent = storage };
                 infra.Add(blobs);
 
-                infra.Add(new ProvisioningOutput("blobs_endpoint", typeof(string)) { Value = storage.PrimaryEndpoints.Value!.BlobUri });
+                infra.Add(new ProvisioningOutput("blobs_endpoint", typeof(string)) { Value = storage.PrimaryEndpoints.BlobUri });
```

Comment:
This was the ugliest remaining part of the API and almost crippled the dev experience in some scenarios.